src/workerd/api/sockets.c++

```diff
@@ -357,7 +357,7 @@ void Socket::handleProxyStatus(jsg::Lock& js, kj::Promise<kj::Maybe<kj::Exceptio
 void Socket::handleProxyError(jsg::Lock& js, kj::Exception e) {
   resolveFulfiller(js, kj::mv(e));
   readable->getController().cancel(js, nullptr).markAsHandled(js);
-  writable->getController().abort(js, nullptr).markAsHandled(js);
+  writable->getController().abort(js, js.v8Error(e.getDescription())).markAsHandled(js);
```

Comment:
Super minor nit: This should work also and would be better in line with current refactoring efforts. js.error(...) returns a `jsg::JsValue` that converts to a `v8::Local<v8::Value>`.

```suggestion
  writable->getController().abort(js, js.error(e.getDescription())).markAsHandled(js);
```

Reply:
Sure, done.